electron-app/src/components/settings/AccountBalances.vue

```diff
@@ -23,7 +31,8 @@
     <v-data-table
       :headers="headers"
       :items="balances"
-      :loading="deleting"
+      :loading="deleting || isLoading"
+      loading-text="Please wait while Rotki fetches your balances..."
```

Comment:
I would rather this said: "Please wait while Rotki queries the blockchain..."

Reply:
Sure give me a moment :)